nginx.conf

```diff
@@ -105,6 +105,9 @@
     rewrite ^/(user/password/send-reset-email)$ $scheme://$http_host/graph/$1;
     rewrite ^/(login)$ $scheme://$http_host/graph/$1; 
 
+    # Remove next line if bug is fixed https://github.com/grafana/grafana/issues/27588
+    rewrite ^/explore(.*)$ /graph/explore$1 permanent;
```

Comment:
It should not be a permanent redirect if we intend to fix in the future.
Please also use the same style as above with $scheme, etc

Reply:
Done!